resources/definitions/ultimaker_method_base.def.json

```diff
@@ -412,9 +416,10 @@
         "support_roof_height": { "value": "4*layer_height if extruder_nr == support_extruder_nr else 5*layer_height" },
         "support_roof_material_flow": { "value": "material_flow" },
         "support_supported_skin_fan_speed": { "value": "cool_fan_speed_max" },
-        "support_top_distance": { "value": "support_z_distance" },
+        "support_top_distance": { "value": "0.25 if extruder_nr == support_extruder_nr else support_z_distance" },
         "support_wall_count": { "value": "2 if support_conical_enabled or support_structure == 'tree' else 0" },
-        "support_xy_distance": { "value": 0.2 },
+        "support_xy_distance": { "value": "0.3 if extruder_nr == support_extruder_nr else 0.2" },
+        "support_xy_distance_overhang": { "value": "0.3 if extruder_nr == support_extruder_nr else 0.2" },
```

Comment:
Similar comment as with support_top_distance.

support_xy_distance & support_xy_distance_overhang are limited to support_infill_extruder_nr